docs/source/examples.rst

```diff
@@ -1,10 +1,295 @@
 .. _examples_page:
 
+########
 Examples
-========
+########
 
-ns-3 simulation
+
+.. _ns3_connection:
+
+*******************
+Connetion with ns-3
+*******************
+
+We will demonstrate the cooperation of Reinforced-lib with external WiFi simulating software based on the example of
+ML controlled rate adaptation manager. To simulate the WiFi environment, we will use the a popular, research oriented
+network simulator - ns-3. To learn more about the simulator, we encourage to visit the
+`official website <https://www.nsnam.org/>`_ or read the
+`ns-3 tutorial <https://www.nsnam.org/docs/release/3.36/tutorial/html/index.html>`_.
+
+
+Environment setup
+=================
+
+In order to perform experiments with pythonic Reinforced-lib and C++ written ns-3, you need to setup the environment in
+following order:
+
+  * Favourite C++ compiler (we assume that you already have one in your dev stack)
+  * ns-3 (connection tested on ns-3.36 version)
```

Comment:
Mamy konkretnie ns-3.36.1, ale to taka drobnostka

Reply:
wiem, ale nie chciałęm tak szczegółowo pisać, bo pewnie wyjdzie zaraz 3.36.2, i już będzie że stare

Reply:
Jasne, rozumiem